tests/numerics/test_time_step_control.py

```diff
@@ -348,6 +348,30 @@ def test_number_of_recomp_attempts_greater_than_zero(
             )
         assert msg in str(excinfo.value)
 
+    @pytest.mark.parametrize("num_time_steps", [10, 20])
+    def test_number_time_steps(self, num_time_steps):
+        """An error should be realised if the number of performed time steps should differ
+        from the effective definition via the time step size. Implicitly, this test also
+        tests whether the time manager correctly detects final times."""
+        time_manager = pp.TimeManager(
+            schedule=[0.0, 1.0],
+            dt_init=1.0 / num_time_steps,
+            constant_dt=True,
+        )
+
+        params = {
+            "time_manager": time_manager,
+            "suppress_export": True,
+        }
+
+        from porepy.models.momentum_balance import MomentumBalance
```

Comment:
Any reason to use momentum balance here, and not a simpler mass_balance?

Reply:
SinglePhaseFlow was included instead of momentum balance.